srcpkgs/phpMyAdmin/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'phpMyAdmin'
+pkgname=phpMyAdmin
+version=4.8.5
+revision=1
+archs=noarch
+wrksrc="${pkgname}-${version}-all-languages"
+conf_files="/etc/webapps/${pkgname}/config.inc.php"
+depends="php mysql"
+short_desc="Web interface for MySQL and MariaDB"
+maintainer="Franc[e]sco <lolisamurai@tfwno.gf>"
+license="GPL-2.0-or-later"
+homepage="https://www.phpmyadmin.net"
+distfiles="https://files.phpmyadmin.net/phpMyAdmin/${version}/phpMyAdmin-${version}-all-languages.tar.xz"
+checksum=d7ab5df4e464d7ba17bf2a42da7d7f26dad45c34bf321ac8ae7d2ed748413913
+
+do_install() {
+	vmkdir usr/share/webapps/${pkgname}
+	cp -rT ${wrksrc} ${DESTDIR}/usr/share/webapps/${pkgname}
```

Comment:
i think vcopy * can be used, no ?

Reply:
oh yeah, i copypasta'd too much from another template